### DISCUSSION TITLE
Is there a way to convert comments to code?

### DISCUSSION THREAD

**Discussion by camsoft2000** [intitni/CopilotForXcode#142] [General]
I know ChatGPT can do this, but Copilot can also do this in VSCode, I tried to create a comment for Copilot to convert to code but instead, it auto-suggested the comment text instead of interpreting it as a prompt.

I'm trying to do something as mentioned in this article https://nira.com/github-copilot/ under "Convert Comments to Code".

Thanks.

**Comment by intitni** [comment]
You can provide more hint to let it know you want code instead of comment, for example,

adding a line break:

```
// a function to generate random number

[wait for a suggestion on this line]
```

give it more context in code
```
// a function to generate random number
func [wait here]
```
